upstream-community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -205,7 +206,700 @@ spec:
                         fieldPath: metadata.namespace
                   - name: TZ
                     value: UTC
-                      
+    
+    # discovery.yaml 
```

Comment:
The CSV does not support adding arbitrary kube resources in the `install` section, so this will fail to parse.